manifests/fastcgi/server.pp

```diff
@@ -32,13 +32,13 @@
 #   Sets a header for the server
 #
 define apache::fastcgi::server (
-  String $host                  = '127.0.0.1:9000',
-  Integer $timeout              = 15,
-  Boolean $flush                = false,
-  String $faux_path             = "/var/www/${name}.fcgi",
-  String $fcgi_alias            = "/${name}.fcgi",
-  String $file_type             = 'application/x-httpd-php',
-  Optional[String] $pass_header = undef,
+  String $host                     = '127.0.0.1:9000',
+  Integer $timeout                 = 15,
+  Boolean $flush                   = false,
+  Stdlib::Absolutepath $faux_path  = "/var/www/${name}.fcgi",
+  Stdlib::Absolutepath $fcgi_alias = "/${name}.fcgi",
```

Comment:
This isn't really an absolute path, it's the path in the URL. That's probably closed to an absolute unix path. It shouldn't accept a Windows path though.

Reply:
I changed to `Stdlib::Unixpath` at several Code-Locations. (next Push)